### ISSUE TITLE
target for BERTopic

### ISSUE THREAD

**Issue by naokityokoyama** [MaartenGr/BERTopic#1300]
Is it possible for me to pass sentences (say 10 sentences) as a target for bertopic???

Exemplo??
fit_transform(X=Corpus, y = [set of 10 sentences])

**Comment by MaartenGr** [comment]
It is currently not possible to set the sentences themselves as target as the `y` will need to be numerical. So if you have numerical values for those sentences (labels), then that might be possible.
